bazel-build.el

```diff
@@ -24,6 +24,17 @@
 ;;
 ;;; Code:
 
+(defgroup bazel-build nil
+  "Package for building and running code using Bazel."
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+(defcustom bazel-build-bazel-command "bazel"
+  "Filename of bazel executable."
```

Comment:
This isn't really a filename since it's used as a program name and searched for in the path. Maybe better something like "Name of the \`bazel' program for \`bazel-build' and `bazel-test'" or so?